src/tools/kata-ctl/src/check.rs

```diff
@@ -100,8 +101,26 @@ pub fn check_cpu_attribs(
     Ok(missing_attribs)
 }
 
-pub fn run_network_checks() -> Result<()> {
-    Ok(())
+#[tokio::main]
+pub async fn run_network_checks() -> Result<()> {
+    println!("Running network checks...");
+    if let Ok((connection, handle, _)) =
+            rtnetlink::new_connection().context("failed to create netlink connection")
+        {
+            let thread_handler = tokio::spawn(connection);
+            defer!({
+                thread_handler.abort();
+            });
+
+            handle
+                .link()
+                .add()
+                .veth("foo".to_string(), "bar".to_string());
```

Comment:
This is not enough IMO. The test can be passed **iff** we check that the pair `foo <-> bar` does exist.

Reply:
you can use `handle.link().get().match_name("your link name").execute();`

Reply:
To generate random string values, you could use one of the following:

- [`UUID::new()`](https://github.com/kata-containers/kata-containers/blob/main/src/libs/kata-sys-util/src/rand/uuid.rs#L21)

  Here are a couple of examples of it's use:

  - https://github.com/kata-containers/kata-containers/blob/main/src/runtime-rs/crates/resource/src/volume/share_fs_volume.rs#L139
  - https://github.com/kata-containers/kata-containers/blob/main/src/runtime-rs/crates/resource/src/network/network_pair.rs#L50

- [`gen_id()`](https://github.com/kata-containers/kata-containers/blob/main/src/runtime-rs/tests/utils/src/lib.rs#L29)

  Although, it might be best if you did want to use this to move it to somewhere like `src/libs/kata-sys-util/`.

Reply:
Thank you for your feedback! I used `UUID::new()` to use random names for the host and VM, and I made sure that I deleted the pairs before I returned. Please, let me know if there are any other changes I need to make. @jodh-intel @amshinde